lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -60,6 +60,15 @@ div.course-wrapper {
       }
     }
 
+    .required-content-msg {
+      margin-bottom: 20px;
+      background-color: $gray-l4;
+      color: $black-t2;
+      font-size: 13px;
```

Comment:
Is there an existing rule in _typography.scss we could use here instead of setting the font-size specifically here? An extend here would be ideal.